blocks/init/src/Blocks/components/menu/menu.php

```diff
@@ -19,20 +19,18 @@
 
 $name = $attributes['menu'] ?? 'header_main_nav';
 $modifier = $attributes['modifier'] ?? '';
+$linkClasses = $attributes['linkClasses'] ?? '';
+$jsModifier = $attributes['jsModifier'] ?? '';
 $variation = isset($attributes['variation']) ? "{$componentClass}-{$attributes['variation']}" : $componentClass;
-$jsClass = $attributes['jsClass'] ?? '';
 
-$parentClasses = Components::classnames([
-	$jsClass ? "js-{$jsClass}" : '',
-]);
-
-$menu = Menu::bemMenu(
+$menu = Menu::bemModifiedMenu(
 	$name,
 	$variation,
-	$parentClasses,
+	$linkClasses,
+	$jsModifier,
 	$modifier ? "{$variation}--{$modifier}" : ''
 );
 
 if (!empty($menu) && !$menu) {
-	echo \esc_html($menu);
+	echo $menu; // phpcs:ignore
```

Comment:
Shouldn't we ignore specific rules here rather than blanket ignore? I.e. `// phpcs:ignore No.Escaping.Rule`

Reply:
Sure. I usually blanket ignore since it's easier in the projects 🤷🏼‍♂️